torch/fx/passes/split_module.py

```diff
@@ -477,6 +477,16 @@ def instantiate_node_partition_mapping(node):
             orig_mod_attr_nodes: List[Node] = [
                 orig_mod_env[key] for key in partition.inputs
             ]
+
+            # We actually need to insert the placeholder nodes in the original order
```

Comment:
test case?

Reply:
The internal diff has a torchrec unittest.